hyq:
  # Publish all joint states -----------------------------------
  # Creates the /joint_states topic necessary in ROS
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 1000

  # Effort Controllers ---------------------------------------
  lf_haa_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lf_haa_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
  lf_hfe_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lf_hfe_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
  lf_kfe_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lf_kfe_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}

    # Effort Controllers ---------------------------------------
  lh_haa_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lh_haa_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
  lh_hfe_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lh_hfe_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
  lh_kfe_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lh_kfe_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}
  
    # Effort Controllers ---------------------------------------
  rf_haa_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: rf_haa_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
  rf_hfe_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: rf_hfe_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
  rf_kfe_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: rf_kfe_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}

    # Effort Controllers ---------------------------------------
  rh_haa_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: rh_haa_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
  rh_hfe_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: rh_hfe_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
  rh_kfe_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: rh_kfe_joint
    pid: {p: 5000.0, i: 0.01, d: 100.0}